crates/sui-protocol-config/src/lib.rs

```diff
@@ -454,6 +524,17 @@ impl ProtocolConfig {
         // To change the values here you must create a new protocol version with the new values!
         match version.0 {
             1 => Self {
+                max_tx_in_batch: Some(10),
+                max_modules_in_publish: Some(8),
```

Comment:
let's make it 64 for now and we may increment even more

Reply:
I am not sure the limit is reasonable to be honest. A package with more than 8 modules seems very doable and there are very good reasons for it. 
Good call on the 100k module limit, however my initial intuition is that we should have a limit on the total size of a user transaction and that is the other dimension we can play with